www/fedp04/cv01/index.php

```diff
@@ -0,0 +1,58 @@
+<?php
+$jmeno = 'Polina';

```

Comment:
Jak řekl mistr Yoda, _"chce to vic informaci, vic datovych typu"_
Např.  namísto `$prace` dat `$availability ` typu boolean, a pak používat ternary operator na to.